app/assets/javascripts/angular/controllers/static_pages.js.erb

```diff
@@ -0,0 +1,57 @@
+app.controller('StaticPagesCtrl', ['$scope', '$http', function($scope, $http){
+
+  $scope.book = {};
+  $scope.form = {};
+
+  $scope.form.create = function(){
+    var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
+
+    var data = {
+      book: $scope.book
+    };
+
+    $http.post(url + '/books', data).success(function(response){
+      console.log(response);
+    });
+  };
+
+  $scope.apiSearch = function(keyEvent, value) {
+    if (keyEvent.which === 13) {
```

Comment:
When user hit ENTER after typing some keywords in the "title" field, the app will send a get request to the api. Then a list of books will be generated for user to pick on. 

ng-submit cannot be used here since it detects '"ENTER" key in the whole form. I dont want the form to be submitted. 

Please see a live demo on http://book-guy.herokuapp.com/#/
Try typing some keywords on the title field then hit ENTER.
This works for the two ISBN fields too.